documentation/restler/MakeTestProxyRecording.md

```diff
@@ -0,0 +1,187 @@
+# How to use RESTler and test-proxy to make traffic recording
+
+In this section, we'll show you how to make traffic recording with RESTler and the Azure SDK test-proxy.
+For information on how to run RESTler, please go to [QuickStart](./QuickStart.md).
+
+The Azure SDK Test-proxy is a tool that provides out-of-process record/playback capabilities compatible with any language.
+It also supports pushing/restoring the API test recordings from a git repository.
+Traffic recordings can provide evidence of running API test, and also help validate that the REST API definition is consistent with service behavior.
+
+## Install the test-proxy
+
+See [how to install test-proxy](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#installation)
+
+## Prepare assets.json file
+
+The assets.json file is a configuration file used by the test-proxy to push/restore recording to/from a git repository.
+Create an assets.json file with following content:
+
+```json
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "",
+  "TagPrefix": "apitest/<ServiceName>/<package>"
+}
+```
+
+Take the appConfiguration data-plane API as an example:
+- ServiceName="appconfiguration"
+- package="dataplane"
+
+So the contents of `assets.json` should look like:
+```json
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "apitest",
```

Comment:
@scbedd requested that we use a non-empty prefix so that REST API recordings are separated from SDK recordings.  I am just complying with his request.